src/main/resources/static/api.yml

```diff
@@ -76,6 +76,40 @@ paths:
         500:
           $ref: '#/components/responses/500Response'
       x-codegen-request-body-name: body
+  /premises/summary:
+    get:
+      tags:
+        - Premises
+      summary: Returns a list of premises
+      parameters:
+        - name: X-Service-Name
+          in: header
+          required: false
+          description: If given, only premises for this service will be returned
+          schema:
+            $ref: '#/components/schemas/ServiceName'
+        - name: X-User-Region
+          in: header
+          required: false
+          description: If given, only premises within this region will be returned
+          schema:
+            type: string
+            format: uuid
```

Comment:
I don't think this header is needed as it's now possible to get the user's probation region without it.